h5grove/content.py

```diff
@@ -91,15 +93,21 @@ def metadata(self, depth=None):
             *super().metadata().items(),
         )
 
-    def data(self, selection: str = None):
+    def data(self, selection: Selection = None):
         if selection is None:
             return self._h5py_entity[()]
 
-        parsed_slice = parse_slice(self._h5py_entity, selection)
+        parsed_slice = (
+            parse_slice(self._h5py_entity, selection)
+            if isinstance(selection, str)
+            else selection
+        )
 
         return self._h5py_entity[parsed_slice]
```

Comment:
```suggestion
        if isinstance(selection, str):
            parsed_slice = parse_slice(self._h5py_entity, selection)
            return self._h5py_entity[parsed_slice]

        return self._h5py_entity[selection]
```

Maybe easier to read this way? :man_shrugging: